src/components/menu/view-mode-toggle-button.jsx

```diff
@@ -53,6 +53,7 @@ export function mapStateToProps(state) {
     textColor: isReportView ? 'black' : '#fafafa',
     buttonText: isReportView ? 'Explore' : 'Report',
     tooltipText: isReportView ? 'Explore this notebook' : 'Go to Report view',
+    backgroundColor: isReportView ? { backgroundColor: '#eee', border: '1px solid #ccc' } : '',
```

Comment:
is `backgroundColor` an appropriate name for a prop that contains more that one style? Do you notice that we're passing we're now passing several props that style one element? What does that suggest about how the code could be simplified?

Reply:
@bcolloran  we can remove both props textColor , backgroundColor and merge them into style props which take care how button is render in reportview and exploreview what you think ?

Reply:
that sounds like a correct simplification @iFlameing -- try to pay attention for opportunities like that to simplify code and consolidate little things that are unnecessarily spread out. Also, always be careful about what you name variables, and make sure the name matches the data the variable holds. Paying careful attention to little details like this will help you to take your coding skills to the next level! :-)

One tip that I try to follow: when you are about to submit a PR, before hitting the 'submit' button, take a few minutes to review the diffs of everything you've changed in the "files changed" tab, and make sure that you haven't missed optimizations like this. Recheck variable names, make sure you don't have extra code you don't need, stuff like that. The diff view will help you to spot your changes and make sure they are what you want

Reply:
These points are very helpful. Thanks for sharing these. I will careful from the next time and implement these points in my further pr. :)